SPEC.md

```diff
@@ -160,14 +163,15 @@ Note: For a CDI file to be valid, at least one entry must be specified in this a
 
 #### OCI Edits
 
-The `containerEdits` field describes edits to be made to the OCI specification. Currently only three kinds of edits can be made to the OCI specification: `devices`, `mounts` and `hooks`.
+The `containerEdits` field describes edits to be made to the OCI specification. Currently only four kinds of edits can be made to the OCI specification: `env`, `devices`, `mounts` and `hooks`.
 
 The `containerEdits` field is referenced in two places in the specification:
   * At the device level, where the edits MUST only be made if the matching device is requested by the container runtime user.
   * At the container level, where the edits MUST be made if any of the of the device defined in the `devices` field are requested.
 
 
 The `containerEdits` field has the following definition:
+  * `env` (array of strings in the format of "VARNAME=VARVALUE", OPTIONAL) describes the environment variables that should be set. These values act as defaults and are merged with any specified when creating a container.
```

Comment:
Small nit we should specify that they appended (i.e: at the end) to the environment array

Reply:
@RenaudWasTaken thank you for the review. I've updated the PR as you've suggested. Please, have a look.